### ISSUE TITLE
Newman run cli-reporter echos "newman"

### ISSUE THREAD

**Issue by stevenscg** [postmanlabs/newman#1588]
The Newman (version 3.9.4 on Linux) cli-reporter seems to output the string "newman" at the start of each collection. I don't recall seeing this behavior with earlier versions of newman.

Is this intentional and necessary? If so, is there a flag to suppress it while retaining other standard output?

We typically loop through directories of collection using bash and "newman run" each of them. This extra "newman" line adds unnecessary noise to the results. 

Example request:
```
newman run /path/to/collection.postman_collection--reporter-no-success-assertions --reporter-no-summary --reporter-no-console
```

Example output:
```
newman

Api - Test

→ 2.1-EXAMPLE
  POST http://10.0.2.15:8091/v1/example [200 OK, 1KB, 311ms]
```

**Comment by shamasis** [comment]
@kunagpal - we can actually have a `--reporter-cli-no-banner`. Trivial.

**Comment by shamasis** [comment]
Added to v4 milestone. @stevenscg - it will take a while - around a month. hope that's okay.

**Comment by kunagpal** [comment]
@stevenscg Thanks for reporting this, the `--reporter-cli-no-banner` option has been added in #1596 and will be available with the next release.

**Comment by stevenscg** [comment]
@shamasis @kunagpal Thanks. Looks like a great solution.

**Comment by kunagpal** [comment]
@stevenscg This has been fixed in Newman v3.10.0 😄

**Comment by stevenscg** [comment]
Thanks @kunagpal!
